opam-client.opam

```diff
@@ -29,7 +29,7 @@ depends: [
   "ocaml" {>= "4.02.3"}
   "opam-state" {= version}
   "opam-solver" {= version}
-  "extlib" {>= "1.7.3" & < "1.7.8"}
+  "base64" {>= "3.1.0"}
```

Comment:
`cudf` depends on extlib, so I think we need to either depend on extlib >= 1.7.8 or conflict extlib < 1.7.8

Reply:
but opam does not depend on it explicitly so removing it from the dependency list is the correct thing to do in my opinion.

Reply:
opam-client _does_ depend on it - it requires extlib either being not present at all or being 1.7.8 or later, as otherwise base64 cannot link. I agree that `depends` feels weird for this, so perhaps `conflicts` is the best, therefore.

It's slightly moot, given that dose3 already has `extlib > 1.7.8`

Reply:
Ah, right. Then a `conflicts` is appropriate in my opinion.

Reply:
There's a similar one still floating around for extlib-compat, although I didn't double-check that that's needed (I think that's because you get a conflict on Windows and possibly macOS over Uchar vs UChar otherwise)

Reply:
Conflicts added.